src/cryptography/utils.py

```diff
@@ -48,18 +49,7 @@ def int_from_bytes(data, byteorder, signed=False):
         assert byteorder == 'big'
         assert not signed
 
-        if len(data) % 4 != 0:
-            data = (b'\x00' * (4 - (len(data) % 4))) + data
-
-        result = 0
-
-        while len(data) > 0:
-            digit, = struct.unpack('>I', data[:4])
-            result = (result << 32) + digit
-            # TODO: this is quadratic in the length of data
-            data = data[4:]
-
-        return result
+        return int(codecs.encode(data, 'hex'), 16)
```

Comment:
This branch isn't used in python3 though, python3 always hits the `int.from_bytes` branch, right?